sk-csharp-chatgpt-plugin/README.md

```diff
@@ -0,0 +1,82 @@
+# Semantic Kernel Azure Functions Starter
+
+The `azure-function` Azure Functions application demonstrates how to execute a semantic function.
```

Comment:
Description can be improved

Reply:
Also can you move this to the root of the project. It will need to be there ad the VS Code extension has a command which references it.

Reply:
Updated. I left the readme where it was since I updated the VS Code extension path to run the function correctly from the root folder.